packages/source-iotsitewise/src/alarms/iotevents/cache.ts

```diff
@@ -0,0 +1,40 @@
+import { AlarmsModelStore, AlarmModel } from './types';
+import { EventsClient } from './client';
+import { IoTEventsToSynchroChartsComparisonOperator } from './constants';
+import { ComparisonOperator } from '@aws-sdk/client-iot-events';
+import { COMPARISON_OPERATOR } from '@synchro-charts/core';
+
+export class Cache {
```

Comment:
all of the other SiteWise models are done in a consistent way within https://github.com/awslabs/iot-app-kit/blob/main/packages/source-iotsitewise/src/asset-modules/sitewise/cache.ts

I wonder if this would actually be simpler by adding this to the `asset-module` (maybe it gets a different name) since the alarm stuff seems to be entangled with various SiteWise entities as is.

Reply:
I have to think about this a bit more. It's true that alarms do depend heavily on assets and asset models.